namespace/type-prefix-registry.md

```diff
@@ -0,0 +1,10 @@
+| Type Prefix | Class Name | Definition | Example | Reference |
```

Comment:
This should be computationally consumable IMO. I don't have a concern with a textual representation but the primary source should be elsewhere (this might be covered by Reece's comment above)

Reply:
Yes machine readable-readable at the minimum, human-readable form may exist in sync with this

Reply:
@jb-adams see [GA4GH YAML in VRS](https://github.com/ga4gh/vrs/blob/main/schema/ga4gh.yaml) for an example of the computable resource we are seeking.